libs/shared/ui/src/lib/components/edition-table/edition-table.tsx

```diff
@@ -0,0 +1,46 @@
+export interface EditionTableCell {
+  content?: React.ReactNode
+  className?: string
+}
+
+export interface EditionTableRow {
+  cells: EditionTableCell[]
+  className?: string
+}
+
+export interface EditionTableProps {
```

Comment:
👍

Reply:
This is indeed a prop, the others are interfaces to me, they are not the props of a component, just the type of a property